static/js/compose_fade.js

```diff
@@ -121,8 +121,8 @@ exports.would_receive_message = function (email) {
     }
```

Comment:
Do we need to also change the `if (email === page_params.email) {` at the top of this function?